binderhub/repoproviders.py

```diff
@@ -263,6 +265,114 @@ def get_build_slug(self):
         return f"zenodo-{self.record_id}"
 
 
+class MecaRepoProvider(RepoProvider):
+    """BinderHub Provider that can handle the contents of a MECA bundle
+
+    Users must provide a spec consisting of a public URL to the bundle
+    The URL origin must be included in the list of allowed_origins when that trait is set
+    """
+
+    name = Unicode("MECA Bundle")
```

Comment:
Following the test error, either `name` should be only `MECA` or `launch.json` changed to contain `MECA Bundle`.